model/src/market/event.rs

```diff
@@ -43,11 +54,22 @@ pub enum RequestorEvent {
         #[serde(rename = "eventDate")]
         event_date: DateTime<Utc>,
         #[serde(rename = "proposal")]
-        proposal: crate::market::Proposal,
+        proposal: Proposal,
+    },
+    #[serde(rename = "ProposalRejectedEvent")]
+    ProposalRejectedEvent {
```

Comment:
To consider:
Since we have `PropsalRejectedEvent`, maybe we could give better name to `ProposalEvent` whose name seems to be to general now. Probably similar to AgreementEvent.

Reply:
Second thing to consider:
I saw in rust Requestor code, that when querying Proposals, he must check if Proposal is initial and make different decisions based on this. Maybe it's reason to separate initial Proposals from counter Proposals??

Reply:
We can't change event names and semantics for `Alpha.3`, because we need to be backward compatible.
And this PR is meant for `Alpha.3`

Reply:
postponed to #53